.github/workflows/image-multiarch.yaml

```diff
@@ -0,0 +1,67 @@
+name: Build & push multi-arch image
+
+on:
+  workflow_call:
+    secrets:
+      GH_TOKEN:
+        description: "GitHub token"
+        required: true
+      ECR_REGISTRY:
+        description: "Repository to push the image"
+        required: true
```

Comment:
Perhaps now that GH supports variables and not only secrets, we could put the `ECR_REGISTRY` as a variable rather than secert?

Reply:
Fixed, introduced new variable `IMAGE_REGISTRY` and removed input for it